music_assistant/controllers/music.py

```diff
@@ -846,16 +847,23 @@ async def set_smart_fades_analysis(
         """Store Smart Fades BPM analysis for a track in db."""
         if not (provider := self.mass.get_provider(provider_instance_id_or_domain)):
             return
-        if analysis.bpm <= 0 or analysis.confidence < 0:
-            # skip invalid values
+        if (
+            analysis.duration <= 0.75 * SMART_CROSSFADE_DURATION
+            or analysis.bpm <= 0
+            or analysis.confidence < 0
+        ):
+            # skip invalid values, we skip analysis that were performed on
+            # a short amount of audio as those are often unreliable
             return
+        beats_json = await asyncio.to_thread(lambda: json.dumps(analysis.beats.tolist()))
```

Comment:
```suggestion
        beats_json = await asyncio.to_thread(lambda: json_dumps(analysis.beats.tolist()))
```
consider using json_dumps from our json util which is based on the (MUCH faster) orjson

Reply:
Done